.pre-commit-config.yaml

```diff
@@ -32,6 +31,11 @@ repos:
         entry: poetry run flake8
         require_serial: true
         types: [python]
+        # These are auto-generated by generateDS and will not pass flake8.
+        # Normally these would be excluded by the 'exclude' filter in
+        # setup.cfg, but this is ignored when files are passed on the command
+        # line. See https://github.com/pre-commit/pre-commit-hooks/issues/112
+        exclude: pywemo/ouimeaux_device/api/xsd/
```

Comment:
This exclude can likely be removed after #376. I added some logic to `disable_code_analyzers` with file level comments.